### PR TITLE
Bring back support for missing labels to legacy cross entropy

### PR DESCRIPTION
**Pull request by danieldk**
The legacy cross-entropy losses were made more strict in v9. However, they are too strict and don't accept missing values anymore.

### CODE REVIEW

File: thinc/legacy/loss.py [hunk 1]
```diff
@@ -65,10 +67,10 @@ def convert_truths(self, truths: TruthsT, guesses: Floats2d) -> Tuple[Floats2d,
                         )
                         raise ValueError(msg)
                     for i, value in enumerate(truths):
-                        if not isinstance(value, str):
+                        if not (isinstance(value, str) or value == None):
```
Comment (kadarakos) [inline thinc/legacy/loss.py:70]:
```suggestion
                        if not (isinstance(value, str) or value is None):
```
Python religion.

Comment (danieldk) [reply thinc/legacy/loss.py]: We discussed this a bit more and the check should be against `missing_value`.

**Review by kadarakos — APPROVED** [review]
